data_safe_haven/resources/workspace/ansible/desired_state.yaml

```diff
@@ -242,13 +242,26 @@
         creates: /etc/audit/rules.d/50-privileged.rules
       notify: Restart auditd
 
-    - name: Copy ClamAV configuration
+    - name: Copy ClamAV daemon configuration
       ansible.builtin.copy:
         src: etc/clamav/clamd.conf
         dest: /etc/clamav/clamd.conf
         mode: '0444'
         owner: clamav
         group: adm
+      register: clamd
+
+    - name: Enable and start ClamAV daemon
```

Comment:
Are we moving this here so that `freshclam` can call the socket after its update?

Reply:
I moved it so the order will be 

1. Write config
2. Ensure daemon is running
3. Restart daemon

and to collect the similar items together.